### ISSUE TITLE
Support for geometry operators

### ISSUE THREAD

**Issue by wolfgangwalther** [PostgREST/postgrest#2088]
### Discussed in https://github.com/PostgREST/postgrest/discussions/2035

<div type='discussions-op-text'>

<sup>Originally posted by **manuganji** November 23, 2021</sup>
Hi, is it possible to use geometry functions for querying. The list is here:

https://www.postgresql.org/docs/current/functions-geometry.html#FUNCTIONS-GEOMETRY-FUNC-TABLE

</div>

**Comment by manuganji** [comment]
I'm not too sure if the operators can be added as the list is really big. But if they are indeed added, I can see ways to use in both select and where clause. For now, I'll try to use via functions/views and see how far that takes me as this looks like a lot of work.

**Comment by steve-chavez** [comment]
Yeah, for now this has to be done via functions/views. I don't know how could we expose so many functions safely.

I think we should do aggregate functions first(https://github.com/PostgREST/postgrest/issues/915#issuecomment-987585190) since there are so few default ones.

---

We also had a request for PostGIS functions on https://github.com/PostgREST/postgrest/issues/223#issuecomment-296901350
